navigator_helpers/llms/gretel.py

```diff
@@ -21,13 +23,16 @@ def completion(
         self,
         model: str,
         messages: list,
-        optional_params: dict[str, Any] = None,
+        optional_params: Optional[dict[str, Any]] = None,
```

Comment:
Thanks for fixing this. I think that `=None` is not necessary and this seems to always be a dict ([code](https://github.com/BerriAI/litellm/blob/16c0307eab601b238f21021b9cba04b6957ad34c/litellm/llms/custom_llm.py#L94)), but we can leave it this way.